templates/repo/issue/milestones.tmpl

```diff
@@ -107,6 +107,23 @@
 							{{.RenderedContent|Str2html}}
 						</div>
 					{{end}}
+					{{if .Labels}}
+						<div class="ui labels list">
+							<span class="labels-list">
+								{{range .Labels}}
+									<a
+										class="ui label item"
+										id="label_{{.ID}}"
+										style="color: white; background-color: {{.Color}}"
```

Comment:
This static white here is incorrect. Check how other labels are rendered. The color should be derived dynamically from the background color.

Reply:
https://github.com/go-gitea/gitea/pull/20620#discussion_r1174495836

I guess `RenderLabel`  `RenderLabels` could handle these colors.